docs/man/git-lfs-faq.adoc

```diff
@@ -0,0 +1,144 @@
+= git-lfs-faq(5)
```

Comment:
I realize that we should probably put this in section 7 of the manual, actually; that's where Git [puts](https://git-scm.com/docs/git#_guides) the `gitfaq(7)` manual page, for instance.

Sorry not to spot this on a first review.

Reply:
No worries.  That's a good thing to note, and I've squashed in a change to that effect.  Once it's green, I'll merge.